beacon_node/beacon_chain/tests/payload_invalidation.rs

```diff
@@ -1222,13 +1221,13 @@ async fn attesting_to_optimistic_head() {
     let get_aggregated = || {
         rig.harness
             .chain
-            .get_aggregated_attestation(attestation.data())
+            .get_aggregated_attestation(&attestation.to_ref())
     };
```

Comment:
Things that accept `&ObjectRef` should be changed to accept `ObjectRef` since `ObjectRef` is already `Copy`. We should make a lint for this at some point..

Reply:
fixed in [7328924](https://github.com/sigp/lighthouse/pull/5810/commits/7328924c5d8eaba5fd72313bb0c2b9447e7893e2)